src/list.tsx

```diff
@@ -53,27 +55,26 @@ interface CommonProps {
     disabled?: boolean;
     withChevron?: boolean;
     'aria-label'?: string;
+    right?: Right;
+    danger?: boolean;
 }
 
-type Right = (({centerY}: {centerY: boolean}) => React.ReactNode) | React.ReactNode;
-
 const renderRight = (right: Right, centerY: boolean) => {
     if (typeof right === 'function') return right?.({centerY});
 
     return centerY ? (
-        <div style={{display: 'flex', alignItems: 'center', height: '100%'}}>{right}</div>
+        <div style={{display: 'flex', flexDirection: 'column', justifyContent: 'center', height: '100%'}}>
```

Comment:
This change fixes [this bug](https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AlCB3AOgOwAS4BcBLAgGxgF5MQAVE86nfAJyIHMALAi4J-XJAD4%2B-AVCIA3QQRgAPArgCMSAPTipI-knXS5CgEyqdm1cLy4AvjhVmcqDH2JlK1Os8bnYAZzCsADsQQ2FQgACIwPv6B2B4s7Fw8Jmai%2BNqSuvJKRumaqToymYZqOeapNnxW2OXYIAA0IAQcMAC2EQgA2iAAshASRF4EAIbMIAC69ehEUI1eHQDM%2BgAMoxZAA)